### ISSUE TITLE
Favorites in Playlist long to start

### ISSUE THREAD

**Issue by Phil995511** [FreeTubeApp/FreeTube#6688]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.

### Describe the bug

Playlist > Your playlists > select one of them but does not start automatically > need to click on it again to finally access the desired content (3 clicks instead of 2)

### Expected Behavior

Start automatically on second selection (second click)

### Issue Labels

accessibility issue, usability issue

### FreeTube Version

v0.23.0 Bêta

### Operating System Version

Debian 12

### Installation Method

Flathub

### Primary API used

Local API

### Nightly Build

- [x] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Please create a screenrecording of this. Start at showing of the Player Settings and then reproducing the issue

**Comment by Phil995511** [comment]
Here are the requested screenshots...

![Image](https://github.com/user-attachments/assets/29dc3819-3b4e-4287-8af6-827153ade742)
![Image](https://github.com/user-attachments/assets/3d01e518-4360-4cde-9dbe-5a5e5c2a1600)
![Image](https://github.com/user-attachments/assets/15c38681-09ed-45a0-8d2f-c4ca4c52cfbb)
![Image](https://github.com/user-attachments/assets/f6d78869-f48e-4389-8795-2426e5fb3d75)
![Image](https://github.com/user-attachments/assets/a3312721-f824-4085-9e6f-119696f51f32)

**Comment by Phil995511** [comment]
![Image](https://github.com/user-attachments/assets/d6aac3fb-33f5-499b-ba66-4a165ce9a6eb)
![Image](https://github.com/user-attachments/assets/448271f2-17f4-4ce1-85d4-c244b4036775)
![Image](https://github.com/user-attachments/assets/1fe9a8ca-7f6a-4ba2-a84d-885d4dd05728)
![Image](https://github.com/user-attachments/assets/4f678c6c-fa19-4f11-976d-e0e596541f6e)
![Image](https://github.com/user-attachments/assets/dcd84d95-dd6a-4de5-a177-7e85644e990a)

**Comment by Phil995511** [comment]
Starting playback of a favorite channel requires 3 clicks and could be reduced to 2 clicks...

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I only requested a screenrecording.

**Comment by Phil995511** [comment]
They are not in the right order, but that's what I did, I only provided you with screenshots showing you as requested a) my custom settings of your application b) the tedious steps to finally start viewing a channel stored in favorites (unfortunately some of these screenshots are not displayed in the right order on github, but if you look at their title, you will find the day and time when these screenshots were taken). Regards

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
There isn't anything useful here except for the player settings. Again i would like to see a screen recording. Also open the dev console ctrl + shift + i before reproducing the issue

**Comment by Phil995511** [comment]
Is this what you want ?

![Image](https://github.com/user-attachments/assets/44c0c0ab-1df9-4564-bb66-17e0f92420d9)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I'm asking for a screen recording so a video. Show me in the video that you open the dev console and then a reproduction of the issue

This issue will be closed when you fail to provide the required info to investigate further

**Comment by absidue** [comment]
We are asking for a screen recording because unless it's something completely different that what we think you mean, it sounds like you are abusing the bug report form to complain about how FreeTube functions, that we won't change and not for something that is actually a bug.

**Comment by Phil995511** [comment]
After thinking about it, it seems to me that we should separate the favorites from the videos to watch later, by creating 2 separate sections, which would allow us to launch the videos to watch more quickly...

Regards.

**Comment by absidue** [comment]
All user playlists are the same, there are no special "favourites" or "watch later" things. Fundamentally this all just boils down to you not wanting to do one more click. Yes, the old favourites thing had one click less, but that was a temporary thing that was never intended to be permanent or last as long as it did, so no we won't be going back to an objectively worse playlists system just because you don't like doing one extra click.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
this is really just a duplicate of https://github.com/FreeTubeApp/FreeTube/issues/1339

if we would make that available than it should probably also apply to your own playlists
